eliot/tests/test_action.py

```diff
@@ -657,6 +657,37 @@ def test_startActionWithParentLogStart(self):
                                   "key": "value"})
 
 
+    def test_startTaskNoLogger(self):
+        """
+        L{startTask} when no logger is given logs to the default ``Logger``.
```

Comment:
:question: Maybe add a comma, or "L{startTask} logs to the default ``Logger`` unless it is supplied with a ``logger`` argument.".